lib/src/native/realm_core.dart

```diff
@@ -809,6 +811,33 @@ class _RealmCore {
   RealmAppCredentialsHandle createAppCredentialsAnonymous() {
     return RealmAppCredentialsHandle._(_realmLib.realm_app_credentials_new_anonymous());
   }
+
+  AppConfigHandle createAppConfig(ApplicationConfiguration configuration, RealmHttpTransportHandle httpTransport) {
```

Comment:
I don't think this needs to be public - we should probably have a `AppHandle createApp(ApplicationConfiguration configuration)` method that internally creates the `AppConfigHandle` and the `SyncConfigHandle`.

Reply:
The method is called `getApp(ApplicationConfiguration configuration)` but is part of the top PR (branch `realm_app`).